File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
king,queen,luk,bishop,night,phone =map(int,input().split())

if king !=1:
  print(1-king, end=' ')
else:
  print(0,end=' ') 

if queen !=1:
  print(1-queen, end=' ')
else:
  print(0,end=' ') 

if luk !=2:
  print(2-luk, end=' ')
else:
  print(0,end=' ') 

if bishop !=2:
  print(2-bishop, end=' ')
else:
  print(0,end=' ') 
  
if night !=2:
  print(2-night, end=' ')
else:
  print(0,end=' ') 

if phone !=8:
  print(8-phone, end=' ')
else:
  print(0,end=' ') 

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.js
const [king, q, r, b, k, p] = require("fs")
	.readFileSync("dev/stdin")
	.toString()
	.split(" ");

console.log(1 - king, 1 - q, 2 - r, 2 - b, 2 - k, 8 - p);
